File: JAVA/Car.java
public class Car {

    // ATRIBUTES AS GLOBAL VARIABLES
    Integer id;
    String license;
    // String driver;
    Account driver;        /*this is an account inheritance*/
    Integer passenger;


    // CONSTRUCTOR METOD
    public Car(String license, Account driver){     /* public Class(mandatory parameters)*/
        this.license = license;     /*this.atribute = parameter*/
        this.driver = driver;
    }


    // METOD to bring the data of this class
    void printDataCar() {
        System.out.println("Car Licence " + license + " Name Driver: " + driver.name);
    }
}

File: JS/Car.js
/* 1)
function Car() {
    this.id;
    this.license;
    this.driver;
    this.passenger;
}
*/

/* 2)
function Car(license, driver) {
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger;
}

Car.prototype.printDataCar = function() {
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}
*/



// USING  EcmaScript 6 STANDAR 
class Car {

    constructor(license, driver) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passenger;
    }

    printDataCar() {
        console.log(this.driver)
        console.log(this.driver.name)
        console.log(this.driver.document)
    }
    
}